### PR TITLE
EPS-612: Improved codebase 

### PR DESCRIPTION
**Pull request by earth9890**
Done sanitisation of URL's and attributes.

### CODE REVIEW

File: timed-content-module/includes/frontend.css.php [hunk 1]
```diff
@@ -18,34 +18,34 @@
...
 
 if ( ! empty( $settings->timed_msg_font ) && 'Default' != $settings->timed_msg_font['family'] ) { ?>
-.fl-node-<?php echo $id; ?> .timed-content-message p {
+.fl-node-<?php echo esc_attr($id); ?> .timed-content-message p {
 	<?php FLBuilderFonts::font_css( $settings->timed_msg_font ); ?>
 }
 <?php } ?>
 
 .fl-node-<?php echo $id; ?> .timed-content-message p {
-	color: #<?php echo $settings->timed_msg_color; ?>;
```
Comment (vijetaR) [inline timed-content-module/includes/frontend.css.php:34]: why is this removed `$settings->timed_msg_color;`

**Review by vijetaR — APPROVED** [review]

File: timed-content-module/includes/frontend.css.php (hunk 1 above)
Comment (earth9890) [reply timed-content-module/includes/frontend.css.php:34]: I'll check and change.

File: readme.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ Contributors: pratikchaskar
 Donate link: https://www.paypal.me/BrainstormForce
 Requires at least: 4.4
 Tags: beaver builder, page builder plugin, timed content, php based timed content
-Stable tag: 1.0.3
+Stable tag: 1.0.4
 Tested up to: 6.5
```
Comment (sushmak02) [inline readme.txt]: @vijetaR WP 6.6 tag is not updated in this PR
Can you get this done??

File: timed-content-module/includes/frontend.css.php [hunk 3]
```diff
@@ -18,34 +18,34 @@
...
-	font-size: <?php echo $settings->timed_msg_size; ?>px;
-	line-height: <?php echo $settings->timed_msg_line_height; ?>px;
-	letter-spacing: <?php echo $settings->timed_msg_letter_spacing; ?>px;
+	color: #<?php echo esc_attr( $settings->timed_msg_color ); ?>;
+	font-size: <?php echo esc_attr( $settings->timed_msg_size ); ?>px;
+	line-height: <?php echo esc_attr( $settings->timed_msg_line_height ); ?>px;
+	letter-spacing: <?php echo esc_attr( ( $settings->timed_msg_letter_spacing ) ); ?>px;
```
Comment (sushmak02) [inline timed-content-module/includes/frontend.css.php]: (( added twice

**Review by sushmak02 — APPROVED** [review]

File: readme.txt (hunk 2 above)
Comment (vijetaR) [reply readme.txt]: @sushmak02 I have raised separate PR for that, we can merge it 
https://github.com/brainstormforce/timed-content-for-beaver-builder/pull/28

File: README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 # Timed Content For Beaver Builder #
 
-**Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce)
+**Contributors:** [pratikchaskar](https://profiles.wordpress.org/pratikchaskar/)
```
Comment (sushmak02) [inline README.md]: Did we made these changes in last update? If we manually added bsf name here in the last update then only make these changes

Comment (vijetaR) [reply README.md]: No, we didn't do any changes in last update. 
It was redirecting to the below earlier: I have reverted to as it is.

`**Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce)`

File: timed-content-module/includes/frontend.php [hunk 5]
```diff
@@ -14,6 +14,6 @@
 		echo Timed_Content_Helper::get_timed_content( $settings );
 	}
 } elseif ( isset( $settings->expire_content_action ) && 'msg' == $settings->expire_content_action && $is_start ) {
-	echo '<' . $settings->timed_tag_selection . ' class="timed-content-message">' . $settings->expire_message . '</' . $settings->timed_tag_selection . '>';
+	echo '<' . esc_attr($settings->timed_tag_selection) . ' class="timed-content-message">' . wp_kses_post( $settings->expire_message ). '</' . esc_attr($settings->timed_tag_selection) . '>';
```
Comment (sushmak02) [inline timed-content-module/includes/frontend.php:17]: Confirm if only esc_attr() will be sufficient to validate the tag selection.
